1.intro/chapter3/customize/notebooks-customize.ipynb

```diff
@@ -0,0 +1,1068 @@
+      "text/plain": [
+       "<Figure size 1122.25x500 with 2 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# load an example dataset\n",
+    "dots = seaborn.load_dataset(\"dots\")\n",
```

Comment:
is data coming from internet? where is dots coming from?

Reply:
yes, i will clarify this.